bench/bench-bytestring.cabal

```diff
@@ -198,3 +198,22 @@ benchmark bench-builder-csv
                     -fdicts-cheap
                     -fspec-constr-count=6
   default-language: Haskell2010
+
+benchmark bench-strict-indices
```

Comment:
Does not build currently. Please declare `c-sources`, similar to other components.
```
  c-sources:        ../cbits/fpstring.c
                    ../cbits/itoa.c
  include-dirs:     ../include
```